src/modules/SystemSettings.rb

```diff
@@ -128,8 +130,13 @@ def Activate
         # set the scheduler
         Bootloader.modify_kernel_params("elevator" => new_elevator)
 
-        # TODO FIXME: set the scheduler for all disk devices,
-        # reboot is required to activate the new scheduler now
+        # activate the scheduler for all disk devices
+        if new_elevator != :missing
```

Comment:
The current default is `cfq` but I do not want to hard code it, it might change in the future.